### DISCUSSION TITLE
Self Hosting: Custom Nuget server

### DISCUSSION THREAD

**Discussion by mieliespoor** [badges/shields#7953] [Q&A] [answered]
We are looking to selfhost and would like to integrate it with an internally hosted nuget server. I don't however see the option to override the nuget base url?

**Comment by chris48s** [comment, marked as answer]
We don't have the ability to set  a base URL for nuget but we could accept a PR for it.
This should not be a big job as we already have the ability to set `apiBaseUrl`. This allows us to share the nuget client code with myget, chocolatey, powershell gallery, etc. I'm going to convert this discussion to an issue.

**Reply by chris48s** [reply]
issue: https://github.com/badges/shields/issues/7954

**Reply by mieliespoor** [reply]
Thanks, I will take a look at how the config is pulled into the services to see if I can submit a PR for this.
